doc/api/http.md

```diff
@@ -128,6 +128,9 @@ added: v0.3.4
     **Default:** `256`.
   * `timeout` {number} Socket timeout in milliseconds.
     This will set the timeout after the socket is connected.
+  * `options` in
```

Comment:
1. It seems this should not be in the list, otherwise, this wrongly seems to be a key, ie `options.options`. Maybe a separate note below?
2. If I am not wrong, docs usually prefer links with bottom references rather than inlined links.